docs/modules/ROOT/pages/index.adoc

```diff
@@ -26,23 +35,20 @@ Developers often use deployment variables, also called environment variables, to
 
 *Note:* There is no functionality to secure sensitive data within a deployment variable. {ProductName} stores deployment variables in a public GitOps repository. Therefore, do not enter any sensitive data in deployment variables at all. 
 
+== Using these documents
+//Come back and clean this up once you have resolved the other PR
 To learn more and to get started with {ProductName}, see the following documents: 
 
 xref:getting-started/index.adoc[Getting started]:: Get started with {ProductName}
 []
-xref:concepts/index.adoc[*Concepts*]:: Learn about technical and conceptual information
```

Comment:
Since all of these blurbs are full sentences, I would add a period at the end of each.